eng/pipelines/runtime.yml

```diff
@@ -283,6 +282,32 @@ jobs:
           eq(dependencies.checkout.outputs['SetPathVars_installer.containsChange'], true),
           eq(variables['isFullMatrix'], true))
 
+#
+# Build the whole product using Mono and run tests
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+    buildConfig: Release
+    runtimeFlavor: mono
+    platforms:
+    - Browser_wasm
+    jobParameters:
+      testGroup: innerloop
+      nameSuffix: AllSubsets_Mono
+      buildArgs: -s mono+libs+installer+libs.tests -c $(_BuildConfig) /p:ArchiveTests=true
+      extraStepsTemplate: /eng/pipelines/libraries/helix.yml
+      extraStepsParameters:
+        creator: dotnet-bot
+        testRunNamePrefixSuffix: Mono_$(_BuildConfig)
```

Comment:
Maybe we should add support to condition the extra steps... I can imagine the tests being executed when there is an installer only change. I think it would only make sense to run wasm tests if libraries or mono tests change, but build if libraries, mono or installer changes.

Reply:
> I think it would only make sense to run wasm tests if libraries or mono tests change, but build if libraries, mono or installer changes.

Not sure if I understood you correctly here, but we do need to run the wasm tests if the libraries or mono implementation changes, not just the tests.

Reply:
What I mean is run the tests if mono or libraries change only, don’t run the tests if only installer is changed. We can do that in a follow up PR though